container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -55,6 +56,16 @@ func CreateEngine(
 		)
 	}
 
+	httpPortSpec, err := port_spec.NewPortSpec(uint16(frontendPortSpec), consts.EngineTransportProtocol, consts.HttpApplicationProtocol, defaultWait)
+	if err != nil {
+		return nil, stacktrace.Propagate(
+			err,
+			"An error occurred creating the engine's http port spec object using number '%v' and protocol '%v'",
+			9711,
+			consts.EngineTransportProtocol.String(),
+		)
+	}
+
 	engineAttrs, err := objAttrsProvider.ForEngineServer(
 		engineGuid,
 		consts.KurtosisInternalContainerGrpcPortId,
```

Comment:
you should add the `httpPortSpec` also here in order to store this number in Docker labels as we do for `privateGrpcPortSpec `

Reply:
good catch; will create a ticket for this and add the label for frontend.